client/src/components/UI/card/JobCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { Card, Avatar, Button } from "antd";
+import { IconExternalLink } from "@tabler/icons-react";
+import styling from "./JobCard.module.css";
+
+interface JobCardProps {
+  name?: string;
+  company_name?: string;
+  company_location?: string;
+  description?: string;
+  avatarUrl?: string;
+  employees?: string;
+  level?: string;
+  match?: string;
+  location?: string;
+  department?: string;
+}
+
+const JobCard: React.FC<JobCardProps> = ({
+  name,
+  company_name,
+  company_location,
+  description,
+  avatarUrl,
+  employees,
+  level,
+  match,
+  location,
+  department,
+}) => {
+  const truncatedDescription: string | undefined = description
+    ? typeof description === "string" && description.length > 370
+      ? `${description.slice(0, 370)}...`
+      : description
+    : undefined;
+
+  return (
+    <Card className={styling.card}>
+      <div className={styling["card-content"]}>
```

Comment:
if you use the css module with `styling`, then all classes should be as the one on the previous line `styling.card`, so this one should be `styling.cardContent` and you should change the name in the corresponding css file (`.card-content` ->`.cardContent`).

Reply:
replaced all